Userland/Games/Hearts/SettingsDialog.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Types.h>
+#include <LibGUI/Dialog.h>
+
+class SettingsDialog: public GUI::Dialog {
+    C_OBJECT(SettingsDialog)
+public:
+    String player_name() const { return m_player_name; }
```

Comment:
```suggestion
    String const& player_name() const { return m_player_name; }
```

Reply:
Correction:
```suggestion
    const String& player_name() const { return m_player_name; }
```

:P

Reply:
Oh, well. Hm. I'm honestly not 100% clear on the difference but I just pushed @gunnarbeutner 's suggestion. Should I change it?

Reply:
Nah, I'm just joking, please use whichever you're comfortable with

Reply:
I should've known!